backend/plugins/frontline_api/src/modules/inbox/graphql/resolvers/customResolvers/index.ts

```
@@ -1,6 +1,7 @@
 import Conversation from './conversation';
 import ConversationMessage from './conversationMessage';
 import Integration from './integration';
+import ConversationMessage from '../conversationMessage';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ConversationMessage'. Consider to delete it or rename it.

'ConversationMessage' is defined here:


(lint/suspicious/noRedeclare)